cse/benchmark.py

```diff
@@ -0,0 +1,103 @@
+import torch
```

Comment:
Should probably remove this top-line folder. You can move it into `benchmarks` if you like.

Reply:
moved